Sources/Networking/Operations/Handling/CustomerInfoResponseHandler.swift

```diff
@@ -32,7 +65,27 @@ class CustomerInfoResponseHandler {
             }
             .mapError(BackendError.networkError)
 
-        completion(result)
+        self.handle(result: result, completion: completion)
+    }
+
+    private func handle(
+        result: Result<CustomerInfo, BackendError>,
+        completion: @escaping CustomerAPI.CustomerInfoResponseHandler
+    ) {
+        guard result.error?.isServerDown == true,
+        #available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *) else {
+            completion(result)
+            return
+        }
+
+        _ = Task<Void, Never> {
+            do {
+                completion(.success(try await self.computeOfflineCustomerInfo()))
```

Comment:
where are we disabling the feature? Meaning, if we merge to main, won't this actually start computing offline entitlements if we get a 5xx? I'm missing the feature flag

Reply:
Done, this is disabled by default now.